tests/python_functional/src/syslog_ng_config/syslog_ng_config.py

```diff
@@ -70,7 +70,7 @@ def set_version(self, version):
         self.__syslog_ng_config["version"] = version
 
     def get_version(self):
-        return self.__syslog_ng_config["version"]
+        return "current"
```

Comment:
I like the opportunity of setting and getting back the version value for the configuration (even if that says `current`). imho it would be better to leave this getter alone, and use the `current` where the `SyslogNgConfig` getting the version information:
https://github.com/syslog-ng/syslog-ng/blob/3d5b1e1a4b62d2f3958e93fb2ea72fb825844117/tests/python_functional/conftest.py#L103-L105

Reply:
Maybe it is better to leave it alone and not touching this part of the code...

Reply:
I am fine with that as well.